### ISSUE TITLE
suggestion to improve faq for waiting app loading

### ISSUE THREAD

**Issue by JohnnyFun** [cypress-io/cypress#3117]
Hi, let me know if this is the wrong place to do this, but I have a suggestion to improve this faq: https://docs.cypress.io/faq/questions/using-cypress-faq.html#How-do-I-wait-for-my-application-to-load

For my app, I often just need to await all my "/api" network requests before continuing, so I wrote a simple command `cy.waitGets()` that seems to work pretty well for me so far:

```
function waitAll(method) {
	cy.server()
	const alias = `all${method.toLowerCase()}s`
	cy.route(method, /\/api/i).as(alias)
	cy.wait('@' + alias)
}

Cypress.Commands.add('waitGets', () => waitAll('GET'))
Cypress.Commands.add('waitPuts', () => waitAll('PUT'))
Cypress.Commands.add('waitPosts', () => waitAll('POST'))
Cypress.Commands.add('waitDels', () => waitAll('DELETE'))
```

So it might help to suggest they use an approach like that there...unless there's a better approach...or unless there's something horribly wrong with this approach that I'm unaware of. 

The caveat, of course, is that it will fail if no network requests happened, so they actually need to place it at points when requests will be sent. But I've found it quicker to write `cy.waitGets()` than to write `cy.server(); cy.route(...).as(alias); cy.wait(@alias);` function for each specific request if you really don't care to assert a specific network request at all.

**Comment by jennifer-shehane** [comment]
Yes, I will move this issue to our [docs](https://github.com/cypress-io/cypress-documentation) repo. 

Also relevant issue within Cypress https://github.com/cypress-io/cypress/issues/1773
